src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -103,10 +104,8 @@ static const std::unordered_map<std::wstring_view, int32_t> vkeyNamePairs {
 // - hstr: the string to parse into a keychord.
 // Return Value:
 // - a newly constructed KeyChord
-KeyChord KeyChordSerialization::FromString(const winrt::hstring& hstr)
+static KeyChord _FromString(const std::wstring& wstr)

```

Comment:
I think if you make this a std::wstring_view, then you can pass the hstring into it directly, without constructing an intermediate wstring